ResearchKitTests/ORKHKSampleTests.m

```diff
@@ -43,46 +46,34 @@ @implementation ORKHKSampleTests
 - (void)setUp
 {
     [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
-}
-
-- (void)tearDown
-{
-    // Put teardown code here. This method is called after the invocation of each test method in the class.
-    [super tearDown];
+    _d1 = [NSDate dateWithTimeIntervalSinceReferenceDate:0];
+    _d2 = [NSDate dateWithTimeInterval:10 sinceDate:_d1];
```

Comment:
This should probably be in `-[ORKHKSampleTests setUp]`, not `-tearDown`. Do the tests still pass with these changes?

Also, IMHO the tests are more readable with `d1` and `d2` local to each test method, not in the `-setUp` method. To each his own, though! :point_up: